bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,453 @@
+
+# Shelly Binding (org.openhab.binding.shelly)
+
+This openHAB 2 Binding implements control for the Shelly series of devices. This includes sending commands to the devices as well as reding the device status and sensor data.
+
+Author: Markus Michels (markus7017)
```

Comment:
Please remove this block. It's covered in the binding.xml. And the documentation does not refer to specific forum posts.

Reply:
done, moved all of this to READMEbeta.md

Reply:
please check new README

To which folder has dev doc to go?

Reply:
done